docs/src/main/sphinx/admin.md

```diff
@@ -21,6 +21,17 @@ admin/fault-tolerant-execution
 (admin-event-listeners)=
 ## Event listeners
 
+Event listeners are plugins that allow streaming of query events to an external
+system. By default, query events are only kept in memory for a limited number of
+queries configured with `query.max-history`, see
+[](/admin/properties-query-management). Only these events are available in the
```

Comment:
The events are not available in the UI.  The UI shows the current state of a query from creation to completion. The events only record start and finish.